packages/components/src/utils.ts

```diff
@@ -0,0 +1,20 @@
+import { colors } from "./theme.js";
+import { Color } from "./types";
+
+export const getDefaultColorString = (color: Color): string => {
+  if (!colors.hasOwnProperty(color)) {
+    throw `Unable to find color`;
+  }
+
+  const colorValue = colors[color];
+
+  if (typeof colorValue === "string") {
+    return colors[color] as string;
```

Comment:
You could return `colorValue` here to avoid casting.

Reply:
Fixed!